packages/block-library/src/list-item/hooks/use-outdent-list-item.js

```diff
@@ -29,93 +30,90 @@ export default function useOutdentListItem( clientId ) {
 		},
 		[ clientId ]
 	);
-	const { replaceBlocks, selectionChange, multiSelect } =
-		useDispatch( blockEditorStore );
+	const {
+		moveBlocksToPosition,
+		removeBlock,
+		insertBlock,
+		updateBlockListSettings,
+	} = useDispatch( blockEditorStore );
 	const {
 		getBlockRootClientId,
-		getBlockAttributes,
-		getBlock,
+		getBlockName,
+		getBlockOrder,
 		getBlockIndex,
-		getSelectionStart,
-		getSelectionEnd,
-		hasMultiSelection,
-		getMultiSelectedBlockClientIds,
+		getSelectedBlockClientIds,
+		getBlock,
+		getBlockListSettings,
 	} = useSelect( blockEditorStore );
 
+	function getParentListItem( id ) {
+		const listId = getBlockRootClientId( id );
+		const parentListItem = getBlockRootClientId( listId );
+		if ( ! parentListItem ) return;
+		if ( getBlockName( parentListItem ) !== listItemName ) return;
+		return parentListItem;
+	}
+
 	return [
 		canOutdent,
-		useCallback( () => {
-			const _hasMultiSelection = hasMultiSelection();
-			const clientIds = _hasMultiSelection
-				? getMultiSelectedBlockClientIds()
-				: [ clientId ];
+		useCallback( ( clientIds = getSelectedBlockClientIds() ) => {
```

Comment:
The hook useOutdentListItem( clientId ) already receives a clientId parameter. It feels strange that the callback the hook returns ignores the parameter and receives a similar parameter can we just make the hook receive clientIds defaulting to getSelectedBlockClientIds and keep the callback without parameters?

Reply:
Not sure I'm following here. The default is to outdent the selected blocks (`clientIds = getSelectedBlockClientIds()`), but we also need the ability to outdent specific blocks for forward Delete (outdent the next block).